src/utils/db.js

```
@@ -0,0 +1,9 @@
+import { Sequelize } from 'sequelize';
+
+export const client = new Sequelize({
+  host: process.env.DB_HOST,
+  username: process.env.DB_USER,
+  password: process.env.DB_PASSWORD,
+  database: process.env.DB_DATABASE,
```

The DB environment variables (DB_HOST, DB_USER, DB_PASSWORD, DB_DATABASE) are used directly. If any are missing the Sequelize instance will be constructed with undefined values and produce confusing runtime errors. Validate required env vars early and throw a clear error so initialization fails fast.